### PR TITLE
More league statistics

### PR DESCRIPTION
**Pull request by Todd-Estes**
-Updated stat_tracker class, stat_tracker_test, and dummy csv's
 
1. winningest_coach method working
1. test valid, ignoring earlier tests until all methods completed (dummy csv changes)

### CODE REVIEW

**Review by the-color-bliu — APPROVED** [review]

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -218,4 +218,33 @@ def lowest_scoring_home_team
     worst_home_scorer["teamName"]
   end
 
+  def winningest_coach(season)
+    coach_game_count = Hash.new(0)
+    coach_wins = Hash.new(0)
```
Comment (ETBassist) [inline lib/stat_tracker.rb:223]: Looks good! We can probably use these default values in the previous methods to cut them down a bit

**Review by ETBassist — APPROVED** [review]
Awesome work!

**Comment by nmking22** [comment]
Merge approved. We'll have to revisit and refactor this at some point, but that is a problem for another day! Also, let's make a note that we may have to add an elsif to handle tie games.

**Comment by the-color-bliu** [comment]
Looking good, let's also focus heavily on refactoring these methods once we get to that point, merge approved!

**Pull request merged**